cookbook/import_export/create-custom-step.rst

```diff
@@ -15,36 +15,88 @@ We will begin by creating a NotifyStep with its configuration and a doExecute me
 .. literalinclude:: ../../src/Acme/Bundle/NotifyConnectorBundle/Step/NotifyStep.php
    :language: php
    :linenos:
-   :lines: 1-4,10-
 
-Create our Step Element
------------------------
+Define the Step as Service
+--------------------------
 
-Then, we implement a Step Element, in our case, a handler responsible for sending a ping request:
+Add your step element to ``steps.yml`` to ensure that it is loaded and processed by ``DependencyInjection/AcmeNotifyConnectorExtension``:
+
+.. literalinclude:: ../../src/Acme/Bundle/NotifyConnectorBundle/Resources/config/steps.yml
+   :language: yaml
+   :linenos:
+
+Configure our new Job
+---------------------
+
+In ``Resources/config/jobs.yml``, we use a first step to export products in csv and we configure the second one to send a notification:
+
+.. literalinclude:: ../../src/Acme/Bundle/NotifyConnectorBundle/Resources/config/jobs.yml
+   :language: yaml
+   :linenos:
+
+Configure the job form provider
+-------------------------------
 
-.. literalinclude:: ../../src/Acme/Bundle/NotifyConnectorBundle/Handler/CurlHandler.php
+In order to setup the form we use a form configuration provider.
+Here we decorate the ``ProductCsvExport`` class in order to retrieve the proper form export configuration.
+Then we add the form configuration for our new ``url`` field.
+
+.. literalinclude:: ../../src/Acme/Bundle/NotifyConnectorBundle/JobParameters/FormConfigurationProvider/ProductCsvExportNotify.php
    :language: php
    :linenos:
-   :lines: 1-3,9-
 
-Define the Step Element as Service
-----------------------------------
+And don't forget to declare this class as a service, with the proper job name as parameter:
 
-Add your step element to ``steps.yml`` to ensure that it is loaded and processed by ``DependencyInjection/AcmeNotifyConnectorExtension``:
+.. literalinclude:: ../../src/Acme/Bundle/NotifyConnectorBundle/Resources/config/job_parameters.yml
+   :language: yaml
+   :linenos:
 
-.. literalinclude:: ../../src/Acme/Bundle/NotifyConnectorBundle/Resources/config/steps.yml
+Configure the job form constraint
```

Comment:
"Configure the job form constraint" -> "Configure the job constraint" not only for the form, used during execution etc